website/docs/reference/snapshot-configs.md

```diff
@@ -301,3 +301,48 @@ snapshots:
 ```
 
 </File>
+
+<VersionBlock firstVersion="1.9">
+
+#### Define column names in snapshot table
+
+Use the following configuration to define the names of the columns created in the [snapshot table](/docs/build/snapshots#snapshot-meta-fields).
+
+<File name='dbt_project.yml'>
+
+```yml
+
+snapshots:
+  my_project:
+    +snapshot_meta_column_names ={
+      dbt_valid_from: example_valid_from,
+      dbt_valid_to: example_valid_to
+    }
+
+```
+
+</File>
+
+- When you run your snapshots, dbt handles the renames from `dbt_valid_from` to `example_valid_from` for you.
+- If you make subsequent updates to the config to rename the column, dbt manages the migration of column names for each configuration change. For example:
```

Comment:
This is not true.

See issue -> https://github.com/dbt-labs/docs.getdbt.com/issues/6112#issuecomment-2374659927

> we aren't doing anything for migration, so if a user renames the columns for a snapshot without updating an existing table, they will get an error. The recommendation is to use these settings for new snapshots, or to arrange to update their existing tables prior to committing a column name change.